### PR TITLE
Donate Page: make buttons a link element and set the link destination

### PR DESCRIPTION
**Pull request by jeremyjo**
- Created `<a>` tags around the 3 "Donate" `<Button>` components
- Set the `<a>` href attribute destination for the 3 "Donate" `<Button>` components to be the Treobytes Paypal Website

### CODE REVIEW

File: src/pages/donate.js [hunk 1]
```diff
@@ -66,7 +66,9 @@ const Donate = ({ componentName }) => {
 								We love our lattes. We also love knowing that our youth are
 								prepared for the future. $5 matters. You matter. Let’s do this.
 							</p>
+							<a href = "https://paypal.me/treobytes" target = "_blank">
```
Comment (DLoftis9) [inline src/pages/donate.js:69]: don't add spaces between the attributes, just write them as 
href="https://paypal.me/treobytes" target="_blank"

you run the risk of getting weird side effects

Comment (DLoftis9) [inline src/pages/donate.js:69]: did you test your code locally? Wrapping the button with an anchor tag alters the width of the button. 
You'll need to update the styles for the donate page and style this anchor tag. You can give the anchor tag(s) the className of "btn-wrapper"

Comment (jeremyjo) [reply src/pages/donate.js:69]: I did test the code locally and saw the width of the button being affected. I am still working to get used to the code and am not able to correct the width of the button yet. So, I was intending to use the smaller width button initially and fix it in a future update. 

I added the className 'btn-wrapper' and am looking to update the styles of the donate page.

Comment (jeremyjo) [reply src/pages/donate.js:69]: I spoke to Ava and she agreed to temporarily go live with the site having the smaller buttons and they can be fixed later.

**Comment by jeremyjo** [comment]
Pushed an updated dev-CT2 that removed the extra spaces on the <a> tag

**Review by DLoftis9 — APPROVED** [review]

**Pull request merged**